### PR TITLE
Fixes xenobotany typo

### PR DESCRIPTION
**Pull request by 9600bauds**
Whoops

### CODE REVIEW

**Comment by clusterfack** [comment]
I warned you about not using defines bro, I WARNED YOU

**Comment by 9600bauds** [comment]
#define THE_POSITION_IN_WHICH_THE_SPREAD_NUMBER_WILL_BE_WHEN_INSIDE_THE_DEVELOPMENT_GENE 3 //this is literally only used twice

**Comment by clusterfack** [comment]
would have been worth it if this didn't break as a result

**Comment by 9600bauds** [comment]
shush and merge it

**Comment by 9600bauds** [comment]
I, the creator of this pull request, at the present time leave written down in public record my personal guarantee that this PR is, as a matter of fact, a concise and necessary hotfix that addresses a current bug or exploitability in the live server. Furthermore, I make promise in this statement that this fix in question has been tested by my own accord, as is my responsibility, and make full guarantee that it functions as promised. I draw to your attention the fact that the bug or exploitability that is addressed by this fix could possibly be potentially urgent in nature. Lastly, I leave up to your personal observation the fact that this particular fix is, by definition, a very small fix with little lines changed, and the nature of both the problem and the fix itself is obvious and readily appreciable to an outside observer.

In conclusion, I plead to any and all collaborators that may see this PR to strongly consider it's immediate merge. At this present time I fully declare myself as the sole responsible for any issues that could possibly transpire from the hasty merge of this PR. Obvious to such consequences would be the loss of credibility to any further pledges of this nature that I could append to my own PRs in the future. However, due to the nature of the fix, as elaborated above, I place my full confidence in that it's immediate merge is in good nature, and that prolonging the time it remains unmerged would only contribute towards extending the longevity of the bug or exploitability that this addresses.

**Comment by Imagebard** [comment]
Jesus Christ how salty

**Comment by ihadtoregisterforthis** [comment]
This should be the norm actually

**Comment by 9600bauds** [comment]
~ I, at the time of writing this, was pretty salty at the fact that several collaborators had looked over this extremely simple and fairly urgent hotfix, yet they chose to not merge it (even after having it pointed out in the IRC), for fairly shitty reasons, which is a stark contrast to many fixes by collaborators, some of which don't even go through the PR process ~

~ However I won't edit out my sentiment that, if Riri thinks that being a useless cunt on purpose is funny, then that ironically makes his whole career as collaborator the joke ~

**Comment by 9600bauds** [comment]
~ I, at the time of writing this, was pretty salty at the fact that several collaborators had looked over this extremely simple and fairly urgent hotfix, yet they chose to not merge it (even after having it pointed out in the IRC), for fairly shitty reasons, which is a stark contrast to many fixes by collaborators, some of which don't even go through the PR process ~

**Comment by 9600bauds** [comment]
ty eggman of coding
